web/backend/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express')
```

Comment:
Codacy has a fix for the issue: [Replace `'express')` with `"express");`](https://app.codacy.com/gh/42ISTblockchain/business-basket-app/pullRequest?prid=9595197)
```suggestion
const express = require("express");
```